playbooks/roles/filebeat/templates/filebeat.yml.j2

```diff
@@ -0,0 +1,43 @@
+filebeat:
+  # List of prospectors to fetch data.
+  prospectors:
+    # Each - is a prospector. Below are the prospector specific configurations
+    -
+      paths:
+        - /var/log/syslog
+        - "/var/log/mail.*"
+        - "/var/log/kern.*"
+        - "/var/log/news.*"
+        - "/var/log/*.emerg"
+        - "{{ elasticsearch_log_dir }}/*"
+        - "{{ nginx_log_dir }}/*.log"
+        - "{{ nginx_log_dir }}/*.err"
+        - "{{ COMMON_LOG_DIR }}/tracking/*.log"
+        - "{{ COMMON_LOG_DIR }}/cms/edx.log"
+        - "/edx/app/edxapp/edx-platform/logs/util1/xqueue/edx.log"
+        - "{{ COMMON_LOG_DIR }}/devpi/supervisor/*"
+        - "/edx/app/edxapp/edx-platform/logs/util1/supervisor/*"
+        - "/edx/app/edxapp/edx-platform/logs/util1/notifier/edx.log"
+        - "/edx/app/edxapp/edx-platform/logs/util1/certs/edx.log"
+        - "/edx/app/edxapp/edx-platform/logs/app1/tracking/tracking.log"
+        - "/edx/app/edxapp/edx-platform/logs/app1/supervisor/*.log"
+        - "/edx/app/edxapp/edx-platform/logs/app1/nginx/*.log"
+        - "/edx/app/edxapp/edx-platform/logs/app1/lms/edx.log"
+        - "/edx/app/edxapp/edx-platform/logs/app1/cms/edx.log"
+        - "{{ COMMON_LOG_DIR }}/ecommerce/*"
+        - "{{ COMMON_LOG_DIR }}/ecommerce_worker/*"
+        - "{{ COMMON_LOG_DIR }}/lms/edx.log"
+        - "{{ logstash_log_dir }}/*.log"
+        - "{{ COMMON_LOG_DIR }}/mongo/*"
```

Comment:
For each of these, we should never include just `*`; that could grab unintended files, probably even processing events twice, once again after the logs are rotated.

The greediest we should ever be is `*.log`, because rotated logs, at least, will have a new extension.
Really though, we should always be as specific as possible. If we know it's only ever `edx.log`, let's use that. But if there are several in the same dir, like supervisor/nginx, we can use `*` if all else fails.